store/errors.go

```diff
@@ -88,3 +90,21 @@ func NewErrNotFound(resource, id string) *ErrNotFound {
 func (e *ErrNotFound) Error() string {
 	return "resource: " + e.resource + " id: " + e.Id
 }
+
+// ErrOutOfBounds indicates that an invalid parameter, usually not related with the entity, was provided
+type ErrOutOfBounds struct {
+	Entity string
+	Args   model.StringInterface
+}
+
+func (e *ErrOutOfBounds) Error() string {
+	msg := "invalid parameter"
+	if len(e.Args) > 1 {
+		msg = "invalid parameters"
```

Comment:
We can just do `msg += "s"`. Or personally I would just write "invalid parameter(s)". But upto you on whichever you prefer.

Reply:
Ok